src/VisualStudio/Core/Def/SymbolSearch/SymbolSearchService.Update.cs

```diff
@@ -96,6 +92,14 @@ private void OnOptionChanged(object sender, EventArgs e)
             }
         }
 
+        private bool IsRegisteredForLanguage(string language)
```

Comment:
Nit: I think I would have just had this do the .Any call inside, since it's already an instance method instead of passing it as a lambda.